releng/release-notes-next/hashdir.feature

```diff
@@ -0,0 +1,7 @@
+A new option `hashdir` has been added to the ccache plugin. Setting it to false
+excludes the build working directory from the hash used to distinguish two
+compilations when generating debuginfo. While this allows the compiler cache
+to be shared across different package NEVRs, it might cause the debuginfo to be
+incorrect.
+The option can be useful when bisecting when the issue bisected does not need
+to be run in a debugger. ([issue#1395)[])
```

Comment:
Nice, thank you for the snippet!  The last sentence is somehow complicated, can we use something like this?
_The option can be used for issue bisecting if running the debugger is unnecessary._

The link has almost invisible typo, use `[issue#1395][]`